src/com/lftechnology/java/training/alina/io/charactercount/CharacterCount.java

```diff
@@ -0,0 +1,40 @@
+package com.lftechnology.java.training.alina.io.charactercount;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Write an example that counts the number of times a particular character, such as e, appears in a file. The character can be specified at
+ * the command line.
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class CharacterCount {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)